b2share/modules/deposit/api.py

```diff
@@ -224,6 +226,25 @@ def submit(self, commit=True):
         if commit:
             self.commit()
 
+    def validate_before_publishing(self):
+        """Common sense validations before submitting. These would be also
```

Comment:
We already do that, the error users get now looks like this: 
````
{ "message": "Validation error.", "errors": [ { "message": "'2a01ee91-36fe-4edb-9734-73d22ac78821' is a required property", "field": "2a01ee91-36fe-4edb-9734-73d22ac78821" } ], "status": 400 }
````

When the field has some semantic meaning (e.g. `open_access`) the error is easy to understand, but for this special case of community specific metadata blocks specified by uuids, the error is confusing.